common/crypto_utils/avalon_crypto_utils/worker_encryption.py

```diff
@@ -308,19 +312,19 @@ def encrypt_work_order_data_json(self, data_objects,
         data_objects.sort(key=lambda x: x['index'])
         i = 0
         for item in data_objects:
-            data = item['data']
+            data = item['data'].encode('UTF-8')
```

Comment:
It's better to take bytes as input (instead of string) and create encrypted bytes. That will keep the function more generic.

Reply:
Done